src/main/ngapp/accio/src/app/post-blobs.service.ts

```diff
@@ -28,20 +28,28 @@ export class PostBlobsService {
    * Fetches the blobUploadURL to post image data to datastore
    */
   private onUpload(formData: FormData) {
+    console.log('here 1');
```

Comment:
If this was just for debugging, then remove it. If you think it might still be useful, then make the message meaningful.